### PR TITLE
Host pattern library on cloud.gov

### PR DESCRIPTION
**Pull request by adborden**
Yay design systems!
https://fec-pattern-library.apps.cloud.gov

- I added continuous deployment through Travis, similar to what we're doing in `fec-style`
- I removed the `docs/` dir which is configured as the build directory, if there was anything in there that shouldn't have been deleted, we should move those into `components/`
- Updated the npm scripts so that `npm run build` will conveniently build everything

### CODE REVIEW

File: manifest.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+path: ./docs
+applications:
+- name: fec-pattern-library
+  memory: 64M
+  buildpack: https://github.com/cloudfoundry/staticfile-buildpack.git
```
Comment (jmcarp) [inline manifest.yml]: Any reason to download the buildpack from github instead of using the cached `staticfile_buildpack`?

**Comment by noahmanger** [comment]
Yesss. Thank you @adborden !

File: package.json [hunk 2]
```diff
@@ -11,10 +11,15 @@
   "devDependencies": {
     "browserify": "^11.0.1",
     "gulp": "3.8.11",
-    "gulp-sass": "^2.3.2"
+    "gulp-sass": "^2.3.2",
+    "npm-run-all": "^4.0.0"
   },
   "scripts": {
+    "build": "npm-run-all --parallel build-sass build-js --serial build-library",
```
Comment (ccostino) [inline package.json]: Nice, thank you!

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  - npm run build
+  - mkdir $HOME/bin
+  - export PATH=$HOME/bin:$PATH
+  - travis_retry curl -L "https://cli.run.pivotal.io/stable?release=linux64-binary&version=6.23.0" | tar xzv -C $HOME/bin
+  - travis_retry cf install-plugin autopilot -f -r CF-Community
+
+deploy:
+  - provider: script
+    skip_cleanup: true
+    script: ./bin/cf_deploy.sh fec-pattern-library fec dev
```
Comment (ccostino) [inline .travis.yml]: Is the intention that we'd only ever deploy to the `dev` space, or will will this be in multiple environments or at least production too?  Perhaps not something we need to worry about now, I'm just curious if we'll have to account for it in the future.

File: manifest.yml (hunk 1 above)
Comment (adborden) [inline manifest.yml]: Oops, @jmcarp is right, I just forgot how to specify the system buildpack. I'll update it.

File: .travis.yml (hunk 3 above)
Comment (adborden) [inline .travis.yml]: I copy/pasted the script from another project which is built to allow configurable deployment to different org/space.

I'm hoping to pull all these scripts out to a CI library so they can be shared better between projects. some day...

**Pull request merged**